utils/keycodes.js

```diff
@@ -38,99 +44,49 @@ export function isMacOS( _window = window ) {
- *
- * @return {string}          The keyboard shortcut.
- */
-export function accessShortcut( character, _isMacOS = isMacOS ) {
-	return keyboardShortcut( accessKeyCode( character.toUpperCase(), _isMacOS ), _isMacOS );
-}
-
-export function accessKeyCode( character, _isMacOS = isMacOS ) {
-	const keyCombo = _isMacOS() ? `${ META }+${ ALT }` : `${ SHIFT }+${ ALT }`;
-	return `${ keyCombo }+${ character }`;
-}
+const modifiers = {
+	primary: ( _isMac ) => _isMac() ? [ COMMAND ] : [ CTRL ],
+	primaryShift: ( _isMac ) => _isMac() ? [ SHIFT, COMMAND ] : [ CTRL, SHIFT ],
+	secondary: ( _isMac ) => _isMac() ? [ SHIFT, ALT, COMMAND ] : [ CTRL, SHIFT, ALT ],
+	access: ( _isMac ) => _isMac() ? [ CTRL, ALT ] : [ SHIFT, ALT ],
+};
 
 /**
- * Create a modifier shortcut based on a single character.
- *
- * This will output Ctrl+G on Windows when "G" is supplied as an argument.
- * This will output Command+G on MacOS when "G" is supplied as an argument.
+ * An object that contains functions to get raw shortcuts.
+ * E.g. rawShortcut.primary( 'm' ) will return 'meta+m' on Mac.
+ * These are intended for user with the KeyboardShortcuts component or TinyMCE.
  *
- * @param {string} character The character for the key command.
- * @param {Object} _isMacOS  isMacOS function by default; used for DI testing.
- *
- * @return {string}          The keyboard shortcut.
+ * @type {Object} Keyed map of functions to raw shortcuts.
  */
-export function primaryShortcut( character, _isMacOS = isMacOS ) {
-	return keyboardShortcut( `${ PRIMARY }+${ character.toUpperCase() }`, _isMacOS );
-}
+export const rawShortcut = mapValues( modifiers, ( modifier ) => {
+	return ( character, _isMac = isMacOS ) => {
+		return [ ...modifier( _isMac ), character.toLowerCase() ].join( '+' );
+	};
+} );
 
 /**
- * Create an access key + primary key shortcut based on a single character.
- *
- * Access key combo is:
- *  - Control+Alt+Command on MacOS.
- *  - Control+Shift+Alt on Windows/everywhere else.
- *
- * @param {string} character The character for the access combination.
- * @param {Object} _isMacOS  isMacOS function by default; used for DI testing.
+ * An object that contains functions to display shortcuts.
+ * E.g. displayShortcut.primary( 'm' ) will return '⌘M' on Mac.
  *
- * @return {string}          The keyboard shortcut.
+ * @type {Object} Keyed map of functions to display shortcuts.
  */
-export function secondaryShortcut( character, _isMacOS = isMacOS ) {
-	return keyboardShortcut( secondaryKeyCode( character.toUpperCase(), _isMacOS ), _isMacOS );
-}
+export const displayShortcut = mapValues( modifiers, ( modifier ) => {
+	return ( character, _isMac = isMacOS ) => {
+		const isMac = _isMac();
+		const replacementKeyMap = {
+			[ ALT ]: isMac ? '⌥option' : 'Alt',
+			[ CTRL ]: isMac ? '⌃control' : 'Ctrl',
+			[ COMMAND ]: '⌘',
+			[ SHIFT ]: isMac ? '⇧shift' : 'Shift',
+		};
+		const shortcut = [
+			...modifier( _isMac ).map( ( key ) => get( replacementKeyMap, key, key ) ),
```

Comment:
You defeated my new ESLint custom rule! (#6247)

Which is to say, I should make this rule more strict 😛